tools/system_libs.py

```diff
@@ -1138,12 +1138,22 @@ class libasan_rt_wasm(SanitizerLibrary):
   src_dir = ['system', 'lib', 'compiler-rt', 'lib', 'asan']
 
 
-class libstandalonewasm(Library):
+class libstandalonewasm(MuslInternalLibrary):
   name = 'libstandalonewasm'
 
   cflags = ['-Os']
   src_dir = ['system', 'lib']
-  src_files = ['standalone_wasm.c']
+
+  def get_files(self):
+    base_files = ['standalone_wasm.c']
+    base_files = [os.path.join('system', 'lib', f) for f in base_files]
+    musl_files = ['strftime.c',
+                  '__month_to_secs.c',
+                  '__tm_to_secs.c',
+                  '__tz.c',
+                  '__year_to_secs.c']
+    musl_files = [os.path.join('system', 'lib', 'libc', 'musl', 'src', 'time', f) for f in musl_files]
```

Comment:
You can use `files_in_path` helper function here.

Reply:
Nice, thanks! Fixing.